src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.Scanner;
+import java.util.List;
```

Comment:
Import statements should be sorted to improve readability. Could try to let the IDE handle this.